core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/cluster/dr/restore/DatabaseRestoreRequest.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.cloudbreak.reactor.api.event.cluster.dr.restore;
+
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.dr.BackupRestoreEvent;
+
+public class DatabaseRestoreRequest extends BackupRestoreEvent {
+
+    public DatabaseRestoreRequest(Long stackId, String backupLocation, String backupId) {
+        super(stackId, backupLocation, backupId);
+    }
+
+    @Override
+    public String selector() {
+        return "DatabaseRestoreRequest";
+    }
```

Comment:
I won't go over all of your requests, but please fix them all.
So you shouldn't do this. If you check `StackEvent` it has already an implementation which will return with the same by default.